docs/Hosts/Spawn-Hosts.md

```diff
@@ -193,75 +164,26 @@ Host <your_persistent_dns_name>
     Hostname <your_persistent_dns_name>
 ```
 
-Q: Can I have my host hibernate instead of shut down for the schedule?
+**Q: Can I have my host hibernate instead of shut down for the schedule?**
 
 A: Evergreen's hosts do not currently support hibernation, they can only shut down. This means that the host will lose
 current machine state (e.g. tmux sessions) when it turns off. Supporting hibernation may eventually be explored as
 future work in [DEVPROD-8579](https://jira.mongodb.org/browse/DEVPROD-8579). In the meantime, if your workflow is
 significantly impacted by the host shutting down and rebooting regularly on a schedule, you can [request a permanent
 exemption](#permanent-exemption).
 
-**Q: How do I choose the time zone when setting a sleep schedule?**
-
-A: You can pick a time zone from the edit host modal by selecting it from the dropdown.
```

Comment:
Do we not want to keep the Q/A about picking a time zone?

Reply:
It's part of the docs for option 1, and users can see it in the UI, so I don't think they would really have questions about it anymore. This question was mostly there as a placeholder when the time zone was always set to their Spruce time zone.